dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsReader.java

```diff
@@ -53,139 +53,79 @@
-                        "   WHEN loc.orientation = 1 THEN -1 ELSE 1 " +
-                        "END AS " + SNP_ORIENTATION_COLUMN +
-                        ",CASE " +
-                        "   WHEN ctg.orient = 1 THEN -1 ELSE 1 " +
-                        "END AS " + CONTIG_ORIENTATION_COLUMN +
-                        ",CASE " +
-                        "   WHEN link.substrand_reversed_flag = 1 THEN -1 ELSE 1 " +
-                        "END AS " + SUBSNP_ORIENTATION_COLUMN +
+                "SELECT " +
+                        SUBSNP_ID_COLUMN +
+                        "," + SUBSNP_ORIENTATION_COLUMN +
+                        "," + REFSNP_ID_COLUMN +
+                        "," + SNP_ORIENTATION_COLUMN +
+                        "," + CONTIG_NAME_COLUMN +
+                        "," + CONTIG_START_COLUMN +
+                        "," + CONTIG_END_COLUMN +
+                        "," + CONTIG_ORIENTATION_COLUMN +
+                        "," + LOC_TYPE_COLUMN +
+                        "," + CHROMOSOME_COLUMN +
+                        "," + CHROMOSOME_START_COLUMN +
+                        "," + CHROMOSOME_END_COLUMN +
+                        "," + REFERENCE_C +
+                        "," + REFERENCE_T +
+                        "," + ALTERNATE +
+                        "," + ALLELES +
+                        "," + HGVS_C_STRING +
+                        "," + HGVS_C_START +
+                        "," + HGVS_C_STOP +
+                        "," + HGVS_C_ORIENTATION +
+                        "," + HGVS_T_STRING +
+                        "," + HGVS_T_START +
+                        "," + HGVS_T_STOP +
+                        "," + HGVS_T_ORIENTATION +
+                        "," + BATCH_COLUMN +
                 " FROM " +
-                        "b" + dbsnpBuild + "_snpcontigloc loc JOIN " +
-                        "b" + dbsnpBuild + "_contiginfo ctg ON ctg.ctg_id = loc.ctg_id JOIN " +
-                        "snpsubsnplink link ON loc.snp_id = link.snp_id JOIN " +
-                        "subsnp sub ON link.subsnp_id = sub.subsnp_id JOIN " +
-                        "batch on sub.batch_id = batch.batch_id JOIN " +
-                        "b" + dbsnpBuild + "_snphgvslink hgvs ON hgvs.snp_link = loc.snp_id JOIN " +
-                        "dbsnp_shared.obsvariation ON obsvariation.var_id = sub.variation_id" +
+                        "dbsnp_variant_load_" + hash(assembly) +
                 " WHERE " +
-                        "batch.batch_id = ? AND " +
-                        "ctg.group_term IN (?) AND " +
-                        "ctg.group_label LIKE ?" +
+                        "batch_id = ? " +
                 " ORDER BY " +
-                        REFSNP_ID_COLUMN;
+                        LOAD_ORDER_COLUMN;
 
         return sql;
     }
 
-    private PreparedStatementSetter buildPreparedStatementSetter(int batch, String assembly, List<String> assemblyTypes) {
+    String hash(String string) {
```

Comment:
Yes, to add a test so that we are generating the proper hash: https://github.com/EBIvariation/eva-tools/pull/51/files/438556fe9402d0b4bc9c7abb67070b8ec1e07160#diff-4b01e3f51d31e961e59098786788a823R121